neuraxle/steps/numpy.py

```diff
@@ -37,34 +37,59 @@ def transform_one(self, data_inputs):
         return data_inputs.flatten()
 
 
-class NumpyConcatenateInnerFeatures(NonFittableMixin, BaseStep):
-    def __init__(self):
+class NumpyConcatenateOnCustomAxis(NonFittableMixin, BaseStep):
+    """
+    Numpy concetenation step where the concatenation is performed along the specified custom axis.
+    :param axis: the axis where the concatenation is performed.
+    :return: NumpyConcatenateOnCustomAxis instance which is inherited by base step.
+    """
+
+    def __init__(self, axis):
+        self.axis = axis
         BaseStep.__init__(self)
         NonFittableMixin.__init__(self)
 
     def transform(self, data_inputs):
+        """
+        Apply the concatenation transformation along the specified axis (specified in constructor).
+        :param data_inputs:
+        :return:
+        Numpy array
+        """
         return self._concat(data_inputs)
 
     def transform_one(self, data_inputs):
+        """
+        Apply the concatenation transformation along the specified axis (specified in constructor)
```

Comment:
I'd say `Apply the concatenation transformation along the specified axis.` For example, in the documentation website, if this method is shown for a class that inherits from it, there is nothing in their constructor and it is pre-specified.

Reply:
Thanks will change it.